README.md

```diff
@@ -1,94 +1,274 @@
+| --prysm                              | start              | Starts Prysm client                                 |
+| --lighthouse                         | start              | Starts Lighthouse client                            |
+| --erigon                             | start              | Starts Erigon client                                |
+| --mainnet                            | init, start, reset | Initializes, starts, or resets LUKSO's mainnet data |
+| --testnet                            | init, start, reset | Initializes, starts, or resets LUKSO's testnet data |
+| --devnet                             | init, start, reset | Initializes, starts, or resets LUKSO's devnet data  |
+| --geth-config                        | start              | Defines the path to TOML config file                |
+| --prysm-config [string]              | start              | Defines the path to the YAML config file            |
+| --geth-bootnodes [string]            | start              | Sets a custom Geth bootnode name                    |
+| --transaction-fee-recipient [string] | start              | Sets the address that receives block fees           |
+| --validator-keys [string]            | start              | Passes the validator keys from a custom directory   |
+| --validator-password [string]        | start              | Passes the assword from a custom directory          |
+| --log-folder [string]                | start              | Sets up a custom log directory                      |
+| --no-slasher                         | start              | Disables slasher                                    |
+| --genesis-json [string]              | start              | Defines the path to genesis JSON file               |
+| --genesis-ssz [string]               | start              | Defines the path to genesis SSZ file                |
+| --deposit-data-json [string]         | validator deposit  | Defines the path to the deposit JSON file           |
+| --gas-price [string]                 | validator deposit  | Defines the gas price in integers as string         |
+| --rpc [string]                       | validator deposit  | Defines the RPC URL on deposit                      |
+| --genesis                            | validator deposit  | Executes deposit to genesis validator contract      |
+| --start-from-index [int]             | validator deposit  | Start deposit from specific block index             |
+| --max-txs-per-block                  | validator          | Maximum amount of txs sent per single block         |
+| --validator-wallet-dir               | validator          | Location of a generated wallet                      |
+| --validator-keys-dir                 | validator          | Path to your validator keys                         |
+| --validator-wallet-password-file     | validator          | Path to your password file                          |
```

Comment:
This is nice, will probably be changed a bit as we update the commands but let's keep it here for now